manifests/kuadrant/kuadrant-operator/base/operator.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     argocd.argoproj.io/sync-wave: "-1"
 spec:
   sourceType: grpc
-  image: quay.io/kuadrant/kuadrant-operator-catalog:v1.0.0-rc4
+  image: quay.io/kuadrant/kuadrant-operator-catalog:v1.0.0
```

Comment:
v1.0.2 is latest

Reply:
Is there any floating tag for the v1.0 catalog that would get automatic updates?

Reply:
Upgraded to v1.0.2 with fab585d. Tested it again and works as expected.